src/actions/index.js

```diff
@@ -141,6 +141,19 @@ export function toggleShowFeatureHints(showFeatureHints) {
   };
 }
 
+export const TOGGLE_SHOW_DATASET_PREVIEWS = 'TOGGLE_SHOW_DATASET_PREVIEWS';
```

Comment:
can we have all the actions related to preferences in 1 file like - `src/actions/preferences.js` as you added preferencesLoaded ?